cms/djangoapps/course_creators/models.py

```diff
@@ -48,6 +48,11 @@ class CourseCreator(models.Model):
                              help_text=_("Current course creator state"))
     note = models.CharField(max_length=512, blank=True, help_text=_("Optional notes about this user (for example, "
                                                                     "why course creation access was denied)"))
+    orgs = models.ManyToManyField(Organization, blank=True,
+                                  help_text=_("Organizations for which content creator is a part of"))
+    all_organizations = models.BooleanField(default=True,
+                                            help_text=_("To restrict the course creation by user in \
+                                                specific organizations"))
```

Comment:
@farhaanbukhsh, it is cleaner to use the implicit string concatenation instead of escaping the new-line character with a `\`. I have also suggested a different help text which starts with an action verb.
```suggestion
                                            help_text=_("Grant the user the permission to create courses "                                                                                                                 
                                                        "in all organizations"))
```